assets/styles/css/styles.css

```diff
@@ -19,74 +28,164 @@ a {
   color: black;
 }
 
-nav {
-  flex: 1 0 auto;
+button {
+  border: none;
 }
 
-footer {
-  flex: 0 0 auto;
+._ibg {
+  position: relative;
 }
 
-input[type="email"] {
+._ibg img {
+  position: absolute;
   width: 100%;
-  height: 25px;
-  border: none;
-  outline: none;
-  font-family: Raleway;
+  height: 100%;
+  top: 0;
+  left: 0;
+  object-fit: cover;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.button {
```

Comment:
It's better to define one class definition and apply each screen resolution in scope of that class name. But i would advice to use mobile first approach as well.

```
.button {
  flex: 0 0 auto;
  width: 200px;
  height: 65px;
  text-transform: uppercase;
  font-weight: 900;
  font-size: 21px;
  line-height: 119%;
  @media screen and (max-width: 768px) {
      width: 130px;
      height: 42px;
      font-size: 13px;
      line-height: 115%;
  }
  @media screen and (max-width: 320px) {
      width: 140px;
      height: 40px;
      font-size: 13px;
      line-height: 115%;
  }
}

```


```
.button {
// styles for mobile device - less than 768px
@media (min-width: 768px) {
// styles that should be overwritten for screen resolution with width more than 768px
}
}
```